app/renderer/components/common/button.js

```diff
@@ -1,51 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
YESSSSSSSS

Reply:
There are normal `<button>` still left, which will be replaced with `BrowserButton` :-)